core/arch/arm/plat-stm32mp1/scmi_server.c

```diff
@@ -179,38 +205,56 @@ static struct stm32_scmi_clk stm32_scmi_clock[] = {
 	CLOCK_CELL(CK_SCMI_MPU, CK_MPU, "ck_mpu", true),
 	CLOCK_CELL(CK_SCMI_AXI, CK_AXI, "ck_axi", true),
 	CLOCK_CELL(CK_SCMI_BSEC, BSEC, "bsec", true),
-	CLOCK_CELL(CK_SCMI_CRYP1, CRYP1, "cryp1", false),
+	CLOCK_CELL_DECPROT(CK_SCMI_CRYP1, CRYP1, "cryp1", false,
```

Comment:
Not needed for clocks as there's apparently nothing malicious at letting a clock enabled.

Reply:
I think it's still better to warn caller (U-Boot/Linux) in case it manipulates a clock it is not expected to.